src/components/IconToggle/IconToggle.st.css

```diff
@@ -0,0 +1,75 @@
+:import {
+    -st-from: "../../../src/tpaStyles/styleableExt";
+    -st-named: wixApply;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-5;
+    DisabledColor: color-3;
+    DefaultFont: "{theme: 'Body-M', size: '12px', lineHeight: '16px'}";
+}
+
+:vars {
+    /*Overrides*/
+    IconColor: --overridable;
+    LabelColor: --overridable;
+    LabelFont: --overridable;
+}
+
+.root {
+    -st-states: checked, disabled, labelPlacement( enum(start, end) );
+    display: inline-block;
+    padding: 0;
+    margin: 0;
+    line-height: 0;
+}
+
+.content {
+    display: flex;
+    align-items: center;
+}
+
+.root:labelPlacement(start) .content {
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+}
+
+.root:labelPlacement(end) .content {
+    flex-direction: row;
+    justify-content: flex-start;
+}
+
+.spacer {
+    width: 6px;
+}
+
+.root:hover .icon {
+    opacity: 0.6;
```

Comment:
Note that this will also apply on disabled mode.
So I guess you should add a rule that says:
```css
.root:disabled:hover .icon {
    opacity: 1;
}
```